package.json

```
@@ -1,37 +1,43 @@
 {
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
+    "ts:check": "tsc",
     "start": "expo start",
     "android": "expo start --android",
     "ios": "expo start --ios",
     "web": "expo start --web",
     "eject": "expo eject"
   },
   "dependencies": {
-    "@react-native-async-storage/async-storage": "~1.17.3",
-    "@react-native-masked-view/masked-view": "0.2.7",
-    "@react-navigation/native": "^6.0.11",
-    "@react-navigation/stack": "^6.2.2",
+    "@expo/webpack-config": "^19.0.0",
+    "@react-native-async-storage/async-storage": "1.18.2",
+    "@react-native-masked-view/masked-view": "^0.2.7",
+    "@react-navigation/native": "^6.1.16",
+    "@react-navigation/stack": "^6.3.28",
     "dotenv": "^10.0.0",
-    "expo": "^46.0.0",
-    "expo-constants": "~13.2.3",
-    "firebase": "9.6.11",
+    "expo": "^49.0.15",
+    "expo-constants": "~14.4.2",
+    "firebase": "^10.5.2",
     "formik": "2.1.4",
-    "react": "18.0.0",
-    "react-dom": "18.0.0",
-    "react-native": "0.69.4",
-    "react-native-gesture-handler": "~2.5.0",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-native": "0.72.10",
+    "react-native-gesture-handler": "2.12.0",
     "react-native-keyboard-aware-scroll-view": "^0.9.4",
-    "react-native-reanimated": "~2.9.1",
-    "react-native-safe-area-context": "4.3.1",
-    "react-native-screens": "~3.15.0",
-    "react-native-web": "~0.18.7",
-    "yup": "0.27.0"
+    "react-native-reanimated": "3.3.0",
+    "react-native-safe-area-context": "4.6.3",
+    "react-native-screens": "3.22.0",
+    "react-native-web": "0.19.10",
+    "yup": "^0.27.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.18.6",
+    "@babel/core": "^7.20.0",
     "@babel/runtime": "^7.9.0",
-    "babel-preset-expo": "~9.2.0"
+    "@types/react": "^18.2.65",
+    "babel-preset-expo": "^9.5.0",
+    "install": "^0.13.0",
+    "npm": "^10.5.0",
```

```suggestion
    "install": "^0.13.0",
    "npm": "^10.5.0",
```

We don't need these dependencies.

---

Why is this resolved when there are no changes pushed to this branch.

---

Oops. Didn't get reflected. Should be pushed now. Also, removed typescript library that I initially added since it wasn't actually used and was experimental in the project.